src/redis-benchmark.c

```diff
@@ -1556,7 +1563,7 @@ int parseOptions(int argc, const char **argv) {
 " -c <clients>       Number of parallel connections (default 50)\n"
 " -n <requests>      Total number of requests (default 100000)\n"
 " -d <size>          Data size of SET/GET value in bytes (default 3)\n"
-" --dbnum <db>       SELECT the specified db number (default 0)\n"
+" --dbnum <db>       SELECT the specified db number (default 0), -1 for random from 0 to 15\n"
```

Comment:
Max of dbnum is not only 15.

Reply:
fix, use -N instead of -1 to incoming the maxDBNum.